ui/index.js

```diff
@@ -36,7 +36,7 @@ export default function launchMetamaskUi(opts, cb) {
   // check if we are unlocked first
   backgroundConnection.getState(function (err, metamaskState) {
     if (err) {
-      cb(err);
+      cb(err, metamaskState);
```

Comment:
As discussed, this callback is expecting a store. Need to modify the callback so that metamaskState can be passed down

Reply:
fixed [here](https://github.com/MetaMask/metamask-extension/pull/14461/commits/23828dfb101c3b9a6ab7ed585f158696edffa91b)

Reply:
so if you take a look at ui/index.js:33 the launchMetamaskUi() function, you'll notice that if there's an error, callback will be called with metamaskState (at this point the store has not been initialised). line42 is where the store is initialised and then the callback is called with null as error..

So basically, if there's an error, store will be = metamaskState

Reply:
oh, I understand now, thanks